### PR TITLE
[Reskin-440] Styles for expenditure component

### PR DESCRIPTION
**Pull request by alejandrocabriales**
## Ticket
https://trello.com/c/ftwQpXA8/440-apply-reskin-design-to-the-cu-index

## What solved
- [X]  Should update the expenditure section with the background and new styles (default/hover). [image.png]

### CODE REVIEW

File: src/stories/components/CuTableColumnTeamMember/CuTableColumnTeamMember.tsx [hunk 1]
```diff
@@ -77,44 +73,59 @@ const Container = styled.div({
...
+  fontSize: '11px',
+
+  color: theme.palette.isLight ? '#434358' : '#9FAFB9',
+  fontWeight: 400,
+  fontStyle: 'normal',
+  marginBottom: '8px',
+  lineHeight: '13px',
+  fontFamily: 'Inter, sans-serif',
+  whiteSpace: 'nowrap',
+  [theme.breakpoints.up('desktop_1194')]: {
```
Comment (YasielCabrera) [inline src/stories/components/CuTableColumnTeamMember/CuTableColumnTeamMember.tsx:128]: This is a legacy breakpoint, we should not use it anymore

File: src/stories/components/CuTableColumnTeamMember/CuTableColumnTeamMember.tsx [hunk 2]
```diff
@@ -77,44 +73,59 @@ const Container = styled.div({
   '@media (min-width: 834px) and (max-width: 1194px)': {
     marginLeft: '0px',
   },
-  [lightTheme.breakpoints.up('desktop_1194')]: {
+  [theme.breakpoints.up('desktop_1194')]: {
```
Comment (YasielCabrera) [inline src/stories/components/CuTableColumnTeamMember/CuTableColumnTeamMember.tsx:76]: This is a legacy breakpoint, we should not use it anymore

File: src/views/CUTable/components/CuTableColumnExpenditures/CuTableColumnExpenditures.stories.tsx [hunk 3]
```diff
@@ -3,11 +3,11 @@ import { CuTableColumnExpenditures } from './CuTableColumnExpenditures';
 import type { Meta } from '@storybook/react';
 
 const meta: Meta<typeof CuTableColumnExpenditures> = {
-  title: 'Components/CUTable/ColumnExpenditures',
+  title: 'Fusion/Components/CoreUnits/ColumnExpenditures',
   component: CuTableColumnExpenditures,
   parameters: {
     chromatic: {
-      viewports: [375, 834, 1194],
+      viewports: [375, 834, 1280],
```
Comment (YasielCabrera) [inline src/views/CUTable/components/CuTableColumnExpenditures/CuTableColumnExpenditures.stories.tsx:10]: `834` is a legacy breakpoint, it should be replaced with `768`

**Review by YasielCabrera — CHANGES_REQUESTED** [review]

File: src/stories/components/CuTableColumnTeamMember/CuTableColumnTeamMember.tsx (hunk 1 above)
Comment (alejandrocabriales) [reply src/stories/components/CuTableColumnTeamMember/CuTableColumnTeamMember.tsx:128]: done

**Review by YasielCabrera — APPROVED** [review]
lgtm

**Pull request merged**